packages/main/src/themes/Input.css

```diff
@@ -44,7 +44,6 @@
 	pointer-events: none;
 	background: var(--sapField_ReadOnly_Background);
 	border-color: var(--sapField_ReadOnly_BorderColor);
-	-webkit-text-fill-color: var(--sapContent_DisabledTextColor);
 	color: var(--sapContent_DisabledTextColor);
```

Comment:
According to the latest design, the text color should be --sapField_TextColor (not --sapContent_DisabledTextColor) in Fiori3 HCW and HCB themes in order to see the text, otherwise it is almost unreadable

Reply:
Done

Reply:
--sapField_TextColor is the default colour, so it is not needed to be set explicitly again.